packages/utilities/carbon.yml

```diff
@@ -0,0 +1,83 @@
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas/carbon-resources.schema.json
+library:
+  id: ibmdotcom-utilities
+  name: IBM.com Utilities
+  description: IBM.com ES6 utility classes
+  externalDocsUrl: https://ibmdotcom-utilities.mybluemix.net
```

Comment:
@kennylam is there a different documentation site than https://ibmdotcom-utilities.mybluemix.net?

Reply:
Should we just use this here too? https://carbon-design-system.github.io/carbon-for-ibm-dotcom/utilities